octobot_trading/util/__init__.py

```diff
@@ -78,3 +81,11 @@ def get_market_pair(config, currency) -> (str, bool):
 def get_reference_market(config) -> str:
     # The reference market is the currency unit of the calculated quantity value
     return get_value_or_default(config[CONFIG_TRADING], CONFIG_TRADER_REFERENCE_MARKET, DEFAULT_REFERENCE_MARKET)
+
+
+def datetime_to_timestamp(date_time, date_time_format):
+    return time.mktime(create_datetime(date_time, date_time_format).timetuple())
+
+
+def create_datetime(date_time, date_time_format):
```

Comment:
Not use I understand the purpose of this function: create a ```datetime.datetime``` from the ```date_time``` argument ? I think we should name the argument differently.